unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore_ECS7.cs

```diff
@@ -32,7 +34,11 @@ public class Hit
         public readonly BaseDictionary<int, BaseRefCountedCollection<object>> pendingSceneResources = new BaseDictionary<int, BaseRefCountedCollection<object>>();
         public readonly BaseDictionary<long, GameObject> shapesReady = new BaseDictionary<long, GameObject>();
         public bool isEcs7Enabled = false;
-        public PointerEvent lastPointerInputEvent = new PointerEvent();
         public RaycastEvent lastPointerRayHit = new RaycastEvent();
+
+        // Should the max buttonId be defined by WebInterface?
+        private const int MAX_BUTTON = (int)WebInterface.ACTION_BUTTON.ACTION_6;
+        public bool[] buttonState = new bool[MAX_BUTTON];
+        public bool[] lastButtonState = new bool[MAX_BUTTON];
```

Comment:
this should be moved to `ECSPointerInputSystem` state, ideally we should use data store only for stuffs that are needed from outside sdk7 responsibility